<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NavigateWidget</class>
 <widget class="QWidget" name="NavigateWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{background-image: url(:/new/prefix1/res/navigate-sensor/navigate-background.png);}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>53</x>
      <y>288</y>
      <width>250</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#groupBox_3{border-image: url(:/new/prefix1/res/navigate-sensor/child-background.png);}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>33</x>
       <y>5</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>船体状态</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>61</x>
       <y>51</y>
       <width>61</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>平衡性:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>61</x>
       <y>75</y>
       <width>61</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>平稳性:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_dip">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>51</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>未获取</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_shk">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>75</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>未获取</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>496</x>
      <y>49</y>
      <width>250</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#groupBox_2{border-image: url(:/new/prefix1/res/navigate-sensor/child-background.png);}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>7</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>三轴加速度</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>61</x>
       <y>42</y>
       <width>24</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>61</x>
       <y>66</y>
       <width>24</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>61</x>
       <y>90</y>
       <width>24</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Z:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_tx">
     <property name="geometry">
      <rect>
       <x>91</x>
       <y>42</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>未获取</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ty">
     <property name="geometry">
      <rect>
       <x>91</x>
       <y>66</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>未获取</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_tz">
     <property name="geometry">
      <rect>
       <x>91</x>
       <y>90</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>未获取</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_Close">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>430</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/res/main/button1.png);</string>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>53</x>
      <y>49</y>
      <width>250</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#groupBox{border-image: url(:/new/prefix1/res/navigate-sensor/child-background.png);}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>31</x>
       <y>7</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>磁场</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>68</x>
       <y>42</y>
       <width>24</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>68</x>
       <y>66</y>
       <width>24</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>68</x>
       <y>90</y>
       <width>24</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Z:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_mx">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>42</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>未获取</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_my">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>66</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>未获取</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_mz">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>未获取</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>496</x>
      <y>288</y>
      <width>250</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#groupBox_4{border-image: url(:/new/prefix1/res/navigate-sensor/child-background.png);}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>39</x>
       <y>6</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>入射光颜色</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ircj">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>未获取</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_34">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string> 三原色成分:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ircj_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>21</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>R:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ircj_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>90</y>
       <width>21</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>G:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ircj_4">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>21</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>B:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ircj_5">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>未获取</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ircj_6">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>110</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>未获取</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>277</x>
      <y>170</y>
      <width>250</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#groupBox_5{border-image: url(:/new/prefix1/res/navigate-sensor/child-background.png);}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>32</x>
       <y>7</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>气象信息</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>61</x>
       <y>31</y>
       <width>45</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>天气:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>61</x>
       <y>55</y>
       <width>45</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>温度:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>61</x>
       <y>79</y>
       <width>45</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>湿度:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>61</x>
       <y>103</y>
       <width>45</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>气压:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_wea">
     <property name="geometry">
      <rect>
       <x>112</x>
       <y>31</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>未获取</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_tem">
     <property name="geometry">
      <rect>
       <x>112</x>
       <y>55</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>未获取</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_hum">
     <property name="geometry">
      <rect>
       <x>112</x>
       <y>79</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>未获取</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_aipr">
     <property name="geometry">
      <rect>
       <x>112</x>
       <y>103</y>
       <width>66</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>未获取</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_28">
    <property name="geometry">
     <rect>
      <x>334</x>
      <y>29</y>
      <width>128</width>
      <height>128</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/navigate-sensor/ship.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
